packages/happychat-connection/src/use-happychat-auth.ts

```diff
@@ -13,14 +12,13 @@ export async function requestHappyChatAuth() {
 	} );
 
 	const url: string = config( 'happychat_url' );
-	const locale = getLocaleSlug();
 
 	const happychatUser: HappychatUser = {
 		signer_user_id: user.ID,
-		locale,
 		groups: [ 'WP.com' ],
 		skills: {
 			product: [ 'WP.com' ],
+			language: [ user.language ],
```

Comment:
Optional: we probably want to do this. Some locales are obscure, like "gl". 

```suggestion
			language: [ getLanguage( user.language ).name ],
```

Reply:
I think HappyChat uses the language code for chat assignment? I thought his helped route chats to HE's who are assigned with that language skill? @klimeryk 

If that is not the case, then you are absolutely right and we should probably make this into a human readable language.

Reply:
Let's compare with inline-help and see what it sends and mimic that.

Reply:
Based on the test - https://github.com/Automattic/happychat/blob/9db3c296ac1f0ca19245d64d03f3f533a872644f/src/service/state/middlewares/system/test/chat-assignment.js

It looks like this is expecting `[ 'en' ]`. I didn't follow the flow to see where it is used specifically though.